src/plugins/intel_gpu/tests/unit/fusions/convolution_fusion_test.cpp

```diff
@@ -3113,6 +3121,15 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp16_prelu_onednn, ::testing::ValuesI
     convolution_test_params{ CASE_CONV_FP16_1, 2, 2, 4 },
 }));
 
+INSTANTIATE_TEST_SUITE_P(fusings_gpu_multi_batch, conv_fp16_prelu_onednn, ::testing::ValuesIn(std::vector<convolution_test_params>{
+    convolution_test_params{ CASE_MUL_BATCH_CONV_FP16_1, 4, 4, 4 },
+    convolution_test_params{ CASE_MUL_BATCH_CONV_FP16_2, 4, 4, 4 },
+    convolution_test_params{ CASE_MUL_BATCH_CONV_U8S8_1, 4, 4, 4 },
+    convolution_test_params{ CASE_MUL_BATCH_CONV_U8S8_2, 4, 4, 4 },
+    convolution_test_params{ CASE_MUL_BATCH_CONV_S8S8_1, 4, 4, 4 },
+    convolution_test_params{ CASE_MUL_BATCH_CONV_S8S8_2, 4, 4, 4 },
```

Comment:
Is it necessary to add such many test cases? Isn't it enough to add just one?

Reply:
Reduced test-cases.